Documentation/doc/Documentation/Installation.txt

```diff
@@ -336,14 +336,14 @@ cmake -DCGAL_HEADER_ONLY=ON . # configure \cgal
 
 and we do not need to run `make` anymore.
 
-\subsubsection subsection_headeronly_without_configuration Header-only without CMake configuration
+\subsubsection subsection_headeronly_without_configuration Header-only without CMake Configuration
 
 Since \cgal 4.12, \cgal can be used in header-only mode, without even
-configuring \cgal libraries. In this case, the program using \cgal
-(example, test, demo, etc.) must be directly configured using CMake. The
-variable `CGAL_DIR` must point to the root directory of the CGAL source
-code (either the root of the unpacked release tarball, or the root of the
-Git repository).
+configuring \cgal\. The program using \cgal (example, test, demo, etc.)
+must be directly configured using CMake in any case, but then \cgal will be
```

Comment:
I would replace "must be" by "can be". It's only an option

Reply:
I would also replace "but then" by ". In this case" ("but" sound likes a drawback)

Reply:
We only document how to use CGAL with CMake, as far as I know.

Maybe later we will document how to use CGAL without CMake, in header-only or not, but I think for now that is out-of-topic.